src/toil/fileStore.py

```diff
@@ -721,6 +721,13 @@ def readGlobalFile(self, fileStoreID, userPath=None, cache=True, mutable=None):
                                                               0.0, False)
         return localFilePath
 
+    def exportFile(self, jobStoreFileID, dstUrl):
+        while jobStoreFileID in self._pendingFileWrites:
```

Comment:
Maybe I'm not understanding the docs, but I don't think that does what I want. I want something along the lines of .waitOnUpload()